controllers/orderController.ts

```diff
@@ -0,0 +1,164 @@
+import { Request, Response } from "express";
+import db from "../models";
+import { error } from "console";
+
+// This method returns all users
+const getAllOrders = async (req: Request, res: Response) => {
+  try {
+    const orders = await db.Order.findAll({
+      attributes: {
+        exclude: ["updatedAt"],
+      },
+      include: [
+        {
+          model: db.Product,
+          as: "products",
+          attributes: ["id", "quantity"],
+        },
+        {
+          model: db.Address,
+          as: "address",
+        },
+      ],
+    });
+
+    if (!orders.length) {
+      res.status(404).json({ error: "No orders found" });
+      return;
+    }
+
+    res.json(
+      orders.map((order: any) => {
+        const plainOrder = order.get({ plain: true });
+        const normalizedOrder = {
+          id: plainOrder.id,
+          userId: plainOrder.userId,
+          products: plainOrder.products.map((product: any) => ({
+            productId: product.id,
+            quantity: product.quantity,
+          })),
+          date: Date.now(),
```

Comment:
I think this returns the current timestamp, where we need to return the created at field which is already stored in milliseconds